src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,213 @@
+        assertEquals(userFirst, expectedUser1);
+        registrationService.register(userSecond);
+        User expectedUser2 = Storage.people.get(1);
+        assertEquals(userSecond, expectedUser2);
+    }
+
+    @Test
+    void register_GetDifferentUsers_Ok() {
+        User userFirst = createUser(22, DEFAULT_PASSWORD_FIRST, DEFAULT_LOGIN_FIRST);
+        User userSecond = createUser(70, DEFAULT_PASSWORD_SECOND, DEFAULT_LOGIN_SECOND);
+        registrationService.register(userFirst);
+        registrationService.register(userSecond);
+        User expectedUser1 = Storage.people.get(0);
+        User expectedUser2 = Storage.people.get(1);
+        assertNotEquals(expectedUser1, expectedUser2);
+    }
+
+    @Test
+    void register_userNullGetExceptionMessage_Ok() {
+        try {
+            registrationService.register(null);
+        } catch (NullPointerException e) {
+            String actualMessage = e.getMessage();
+            String expectedMessage = "User should not be null";
+            assertEquals(actualMessage, expectedMessage);
+        }
+    }
+
+    @Test
+    void register_userOneDoubleExceptionMessage_Ok() {
+        User userFirst = createUser(22, DEFAULT_PASSWORD_FIRST, DEFAULT_LOGIN_FIRST);
+        registrationService.register(userFirst);
+        User userDoubleFirst = createUser(22, DEFAULT_PASSWORD_FIRST, DEFAULT_LOGIN_FIRST);
+        try {
+            registrationService.register(userDoubleFirst);
+        } catch (RuntimeException e) {
+            String actualMessage = e.getMessage();
+            String expectedMessage = "User should have unique login";
+            assertEquals(actualMessage, expectedMessage);
+        }
+    }
+
+    @Test
+    void register_userTwoDoubleException_notOk() {
+        User userFirst = createUser(22, DEFAULT_PASSWORD_FIRST, DEFAULT_LOGIN_FIRST);
+        User userSecond = createUser(70, DEFAULT_PASSWORD_SECOND, DEFAULT_LOGIN_SECOND);
+        registrationService.register(userFirst);
+        registrationService.register(userSecond);
+        User userDoubleSecond = createUser(70, DEFAULT_PASSWORD_SECOND, DEFAULT_LOGIN_SECOND);
+        assertThrows(RuntimeException.class, () -> registrationService.register(userDoubleSecond));
+    }
+
+    @Test
+    void register_userTooYoungException_notOk() {
+        User userTooYoung = createUser(10, DEFAULT_PASSWORD_FIRST, DEFAULT_LOGIN_FIRST);
+        assertThrows(RuntimeException.class, () -> registrationService.register(userTooYoung));
+    }
+
+    @Test
+    void register_userTooYoungExceptionMessage_Ok() {
+        User userTooYoung = createUser(10, DEFAULT_PASSWORD_FIRST, DEFAULT_LOGIN_FIRST);
+        try {
+            registrationService.register(userTooYoung);
+        } catch (RuntimeException e) {
+            String actualMessage = e.getMessage();
+            String expectedMessage = "User too young!";
+            assertEquals(actualMessage, expectedMessage);
+        }
+    }
+
```

Comment:
one from these methods is redundant, you don't need check one case several times and use values closer to valid values, check another methods